python/sdist/amici/petab/cli/import_petab.py

```diff
@@ -102,18 +115,42 @@ def _parse_cli_args():
     )
 
     args = parser.parse_args()
-
-    if not args.yaml_file_name and not all(
+    if any(
+        [
+            args.sbml_file_name,
```

Comment:
parser doesn't have them as options?

Reply:
It does, all arguments from `parser.add_argument_group().add_argument()` will be available through that  `parser`.

Reply:
ah :)